### PR TITLE
{tools}[GCCcore/11.3.0] task-spooler v1.0.2

### PR DESCRIPTION
**Pull request by ItIsI-Orient**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by ItIsI-Orient** [comment]
Test report by @ItIsI-Orient
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3703.slaking.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 3.6.8
See https://gist.github.com/16a13b004c1439e03811625ca5c75869 for a full test report.

File: easybuild/easyconfigs/t/task-spooler/task-spooler-1.0.2-GCCcore-11.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+
+source_urls = [homepage]
+sources = ['ts-%(version)s.tar.gz']
+checksums = ['f73452aed80e2f9a7764883e9353aa7f40e65d3c199ad1f3be60fd58b58eafec']
+
+dependencies = {
+    ('binutils', '2.38'),
+}
```
Comment (bedroge) [inline easybuild/easyconfigs/t/task-spooler/task-spooler-1.0.2-GCCcore-11.3.0.eb:17]: `binutils` is only needed for building:
```suggestion
builddependencies = [('binutils', '2.38')]
```

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
bob-Latitude-5300 - Linux Ubuntu 22.04, x86_64, Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz, Python 3.10.6
See https://gist.github.com/bc9fa3aaee01c0ece057a2cd031f42d1 for a full test report.

**Comment by bedroge** [comment]
@boegelbot please test @ generoso

**Comment by bedroge** [comment]
Test report by @bedroge
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
pg-interactive.hpc.rug.nl - Linux CentOS Linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2680 v3 @ 2.50GHz, Python 3.6.8
See https://gist.github.com/495d8f9b30e578cb9bd3cb28ef1e4f89 for a full test report.

**Review by bedroge — APPROVED** [review]
lgtm

**Comment by bedroge** [comment]
Going in, thanks @ItIsI-Orient!

**Pull request merged**